src/axom/inlet/Table.cpp

```diff
@@ -44,7 +44,7 @@ std::shared_ptr<Table> Table::addTable(const std::string& name,
     found = currName.find("/");
   }
 
-  std::string currTableName =
+  const auto currTableName =
```

Comment:
I kinda like leaving this `std::string` instead of `auto`, helps with readability.